rchain/client.py

```diff
@@ -184,6 +184,10 @@ def get_data_at_name(self, par: Par, depth: int = -1) -> Data:
         wrapped = response.payload
         return Data.FromString(wrapped.SerializeToString())
 
+    def get_data_at_par(self, par: Par, block_hash: str, use_pre_state_hash: bool) -> RhoDataResponse:
+        query = DataAtNameByBlockQuery(par=par, blockHash=block_hash, usePreStateHash=use_pre_state_hash)
+        return self._deploy_stub.getDataAtName(query)
```

Comment:
The check for additional errors is missing - `self._check_response`.

_The reason for this additional error checking is wrong implementation for error handling in RNode gRPC service definitions with `ServiceError` type._

Reply:
Yes, good point. Fixed in [82fa891](https://github.com/rchain/pyrchain/pull/81/commits/82fa8916f477de73528d873b5fa6328d74da6d8a).